client/sharkey_client.go

```diff
@@ -237,3 +216,21 @@ func buildConfig(caBundlePath string) (*tls.Config, error) {
 		MinVersion: tls.VersionTLS12,
 	}, nil
 }
+
+func (c *context) shellOut(command []string) {
+	if c.conf.Sudo {
+		command = append([]string{"/usr/bin/sudo"}, command...)
+	}
+	cmd := exec.Cmd{
+		Path: command[0],
+		Args: command,
+	}
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+	err := cmd.Run()
```

Comment:
I think "mv" and "sudo" both print errors stderr.

We could then just print ExitError.Stderr.